src/gt4py/next/program_processors/runners/dace_iterator/__init__.py

```diff
@@ -307,3 +312,177 @@ def build_sdfg_from_itir(
         sdfg.save(sdfg_filename)
 
     return sdfg
+
+
+@dataclasses.dataclass(frozen=True)
+class Program(decorator.Program, dace.frontend.python.common.SDFGConvertible):
+    """Extension of GT4Py Program implementing the SDFGConvertible interface."""
+
+    sdfg_closure_vars: dict[str, Any] = field(default_factory=dict)
+
+    # Defined as class variable, otherwise the same connectivity table -name & data descriptor- across multiple GT4Py Programs
+    # will be considered as a different table (with subsequent name mangling). Therefore, the memory address of the table's data descriptor needs to be the same.
+    # This affects the case where a DaCe.Program calls multiple GT4Py Programs (fused SDFG).
```

Comment:
You are right. The comment is unnecessary long and confusing. This choice about class variable is not at all restrictive, since the connectivitiy tables are defined fully symbolically. So multiple GT4Py programs would just need to set/concretize the symbols in their own run. The detail about dace data descriptors to share the same memory address -to avoid name mangling- is indeed very DaCe specific and not needed at this point.